### PR TITLE
Finish basic install role

### PR DESCRIPTION
**Pull request by aidenkeating**
This will create the configmap requried to configure the Kubernetes Plugin in Jenkins. Currently it points to a node6 slave image to support Ionic 2 builds instead of the default slave image which uses node 4.

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
   console.openshift.io/iconClass: font-icon icon-jenkins
 plans:
   - name: default
-    description: Example description
+    description: Mobile focused CI CD service that integrates with Jenkins Pipelines to test and build your mobile binaries.
```
Comment (david-martin) [inline apb.yml]: @aidenkeating  can we add a label/tag to make it show up in the 'Mobile > Services' sub-category, and the CI/CD (Jenkins) category?

File: apb.yml [hunk 2]
```diff
@@ -19,4 +19,3 @@ plans:
       description: "You need to accept the Android Software Development Kit License agreement in order to continue."
       title: Android SDK License Agreement
       required: True
```
Comment (david-martin) [inline apb.yml]: @aidenkeating I was able to leave the box unchecked and click 'Create'. It looks like nothing got provisioned, but the UI made it seem like it went ahead and provisioned it.

File: apb.yml [hunk 3]
```diff
@@ -19,4 +19,3 @@ plans:
       description: "You need to accept the Android Software Development Kit License agreement in order to continue."
```
Comment (david-martin) [inline apb.yml]: @aidenkeating Is it possible to have a link to the license?

Comment (aidenkeating) [reply apb.yml]: There should be a license on the left description, we can place it here also though if needed.

Comment (david-martin) [reply apb.yml]: I missed the link on the left. So adding the link inline might be a good thing.
The left text also gets trucated (with ...) right at the end of the link, which makes it look like the link might be shortened
image

**Comment by david-martin** [comment]
@aidenkeating Can we use 'aerogear-digger' as the service name, and 'aerogear-digger-apb' as the image name?
The ASB only allows pulling images from 1 repository (that I'm aware of), so we'll still have to push to the feedhenry docker hub org. This is OK.

**Comment by david-martin** [comment]
@aidenkeating A run through locally works great :+1: 
Some comments inline, and a linked PR in mcp for updating the expected service name.

I'd say we can go back to using the upstream slave image as it has node6 support added recently https://github.com/aerogear/digger-android-slave-image/pull/4

**Comment by aidenkeating** [comment]
@david-martin Would you mind taking a look? I'm not sure whether the UI will update automatically, but it's now a text field that pattern matches for `yes`, if `yes` is not typed into the field then the Create button will be disabled.

Also updated to `aerogear-digger`, until I delete the `buildfarm-apb` repo on Docker Hub there may be two Mobile CI/CD's, the newer one has the text field instead of a check box

**Pull request merged**